### DISCUSSION TITLE
Nvchad with vim

### DISCUSSION THREAD

**Discussion by System3-2** [NvChad/NvChad#2039] [General]
is it possible to use nvchad with vim and not neovim

**Comment by siduck** [comment]
@OlojaMuyiwa  no :(

**Reply by System3-2** [reply]
oh okay

**Reply by siduck** [reply]
@OlojaMuyiwa yep its not possible, neovim has all the modern stuff since  v0.5 with lua and all. What makes you not use neovim ?

**Reply by System3-2** [reply]
i already have lazyvim installed i'm trying to see if nvchad is better.

**Reply by System3-2** [reply]
i guess i'll just have to install it then
